server/core/storelimit/limit.go

```diff
@@ -22,16 +22,28 @@ const (
 	AddPeer Type = iota
 	// RemovePeer indicates the type of store limit that limits the removing peer rate
 	RemovePeer
-
 	storeLimitTypeLen
 )
 
+// PriorityLevel lower level means higher priority
+type PriorityLevel int
```

Comment:
It's a little bit strange to me because it actually describes the priority of the operator but is defined in the store limit package.